Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragment.kt

```diff
@@ -118,6 +119,16 @@ class HomeFragment : Fragment(R.layout.home_fragment_layout), AutoInject {
             }
         }
 
+        // Usage example of the new new builder
+        createDialog {
+            title("New implementation")
+            message("Putting the app in background won't cause a crash with the dialog open")
+            positiveButton("Ok") { dummyDialog1() }
+            negativeButton(R.id.button_done)
```

Comment:
You mean R.string... though?

Reply:
Ah yes, I just needed an id of something real quick :D

Reply:
After adding the annotation this display an error to indicate it is not a string resource 👍🏼